persist-mongodb/src/main/kotlin/io/saagie/updatarium/persist/MongodbPersistEngine.kt

```diff
@@ -29,28 +29,31 @@ import org.litote.kmongo.*
 import java.time.Instant
 
 const val MONGODB_PERSIST_CONNECTIONSTRING = "MONGODB_PERSIST_CONNECTIONSTRING"
+const val MONGODB_PERSIST_DATABASE = "MONGODB_PERSIST_DATABASE"
+const val MONGODB_PERSIST_CHANGESET_COLLECTION = "MONGODB_PERSIST_CHANGESET_COLLECTION"
 const val DATABASE = "Updatarium"
 const val COLLECTION = "changeset"
 
 class MongodbPersistEngine(override val configuration: PersistConfig = PersistConfig()) : PersistEngine(configuration) {
 
     private val collection by lazy {
         with(KMongo.createClient(ConnectionString(getConnectionString()))) {
-            this.getDatabase(DATABASE).getCollection<MongoDbChangeSet>(COLLECTION).apply {
-                this.createIndex(
-                    Indexes.ascending("changeSetId"),
-                    IndexOptions().name("changeSetId_1")
-                )
-            }
+            this.getDatabase(getEnvOrDefault(MONGODB_PERSIST_DATABASE, DATABASE))
+                .getCollection<MongoDbChangeSet>(getEnvOrDefault(MONGODB_PERSIST_CHANGESET_COLLECTION, COLLECTION))
+                .apply {
+                    this.createIndex(
+                        Indexes.ascending("changeSetId"),
+                        IndexOptions().name("changeSetId_1")
+                    )
+                }
         }
     }
 
-    private fun getConnectionString(): String {
-        if (System.getenv().containsKey(MONGODB_PERSIST_CONNECTIONSTRING)) {
-            return System.getenv(MONGODB_PERSIST_CONNECTIONSTRING)
-        }
-        throw IllegalArgumentException("$MONGODB_PERSIST_CONNECTIONSTRING is missing (environment variable)")
-    }
+    private fun getEnvOrDefault(name: String, default: String) = System.getenv().getOrDefault(name, default)
```

Comment:
Not sure this function is necessary ... same signature ... same behavior...

Reply:
done